server/images.go

```diff
@@ -1005,6 +1005,39 @@ func PruneLayers() error {
 	return nil
 }
 
+func PruneDirectory(path string) error {
+	info, err := os.Stat(path)
+	if err != nil {
+		return err
+	}
```

Comment:
```suggestion
	info, err := os.Lstat(path)
	if err != nil {
		return err
	}

	// Skip symlinks to prevent infinite loops
	if info.Mode()&os.ModeSymlink != 0 {
		return nil
	}
```

nit to prevent loops in the unlikely case there is a symlink in the directory